src/ontology/cl-edit.owl

```diff
@@ -28147,6 +28148,20 @@ SubClassOf(obo:CL_4023071 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_1009
 SubClassOf(obo:CL_4023071 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023071 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P09240))
 
+# Class: obo:CL_4023075 (L6 tyrosine hydroxylase positive Sst expressing Pvalb-like GABAergic cortical interneuron (Mus musculus))
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22357664") obo:IAO_0000115 obo:CL_4023075 "A Sst GABAergic cortical interneuron found in L6 that expresses tyrosine hydroxylase and is transcriptomically related to Pvalb interneurons. L6 Th+ SST/PV cells have mostly local axonal arborization within L6.")
```

Comment:
@dosumis just to double check before merging anything, is "transcriptomically related" ok for CL? Thanks